packages/codegen-ui-react/lib/__tests__/studio-ui-codegen-react.test.ts

```diff
@@ -170,6 +170,41 @@ describe('amplify render tests', () => {
       expect(componentText).toContain(`key={\`\${item.name}\${item.description}\`}`);
       expect(componentText).toMatchSnapshot();
     });
+
+    describe('GraphQL', () => {
```

Comment:
Are declaration files worth testing here?

Reply:
I saw there were no declaration file testing for collections, so didn't add tests for the new GraphQL tests.